server/controllers/client.ts

```
@@ -11,7 +11,7 @@ const clientControllers: IClientController = {
 
   async render(ctx) {
     const { params, query = {} } = ctx;
-    const { type, menu: menuOnly, path: rootPath, locale } = query;
+    const { type, menu: menuOnly, path: rootPath, locale, populate } = query;
```

IMO we should validate `populate` so if we get garbage we can fail fast.